object/interp.go

```diff
@@ -0,0 +1,36 @@
+package object
+
+import (
+	"errors"
+)
+
+var commands map[string]Extension
+
+func init() {
+	Init()
+}
+
+func Init() {
+	commands = make(map[string]Extension)
+}
+
+func CreateCommand(cmd Extension) error {
+	if cmd.Name == "" {
+		return errors.New("empty command name")
+	}
+	if cmd.MaxArgs != -1 && cmd.MinArgs > cmd.MaxArgs {
+		return errors.New(cmd.Name + ": min args > max args")
```

Comment:
Come on 😅


```suggestion
		return fmt.Errorf("%s: min args > max args", cmd.Name)
```

(Here and in a few other place)

Reply:
perfprint linter?

I tend to use sprintf way for readability purpose, I hated this pattern a few years ago. Not it seems natural, especially as it helps with i18n. But it's not the point here

Reply:
> perfprint linter?

yeah probably, and I did change some in fortio when that linter first showed up

I added the cmd.Name because when it was panic() you could tell which command was wrong, if any, but now that it just bubbles up as error, the cmd.Name is needed - if I had started from fmt.Errorf (which I kinda did and then a linter told me to switch to `errors.New`...) I would agree but, it'll stay like this for now :)